### PR TITLE
Autorun blocks, no autorun js

### PR DESCRIPTION
**Pull request by pelikhan**
- [x] start blocks, auto-run
- [x] start js, no auto-run
- [x] swtich to blocks, auto-run
- [x] switch to js, no auto-run

Fix for https://github.com/microsoft/pxt-arcade/issues/1281

### CODE REVIEW

File: webapp/src/blocks.tsx [hunk 1]
```diff
@@ -162,8 +162,10 @@ export class Editor extends toolboxeditor.ToolboxEditor {
         }
     }
 
-    isDropdownDivVisible() {
-        return Blockly && Blockly.DropDownDiv && Blockly.DropDownDiv.isVisible();
+    isDropdownDivVisible(): boolean {
+        return typeof Blockly !== "undefined"
```
Comment (pelikhan) [inline webapp/src/blocks.tsx]: "blockly is undefined"

**Comment by abchatra** [comment]
This change only impacts arcade right? All other targets autorun is on by default in both javascript & blocks

**Comment by pelikhan** [comment]
This impacts all targets.

________________________________

**Comment by abchatra** [comment]
Ok thinking it again, it makes sense. Although what is the meaning of 'autorun' in pxttarget now? 
https://github.com/microsoft/pxt-arcade/blob/master/pxtarget.json#L265

**Comment by pelikhan** [comment]
it controls whether blocks autoRun or not. JS never autoruns.

**Pull request merged**